src/test/kotlin/de/novatec/betting/game/teams/TeamsServiceUnitTest.kt

```diff
@@ -0,0 +1,39 @@
+package de.novatec.betting.game.teams
+
+import de.novatec.betting.game.openliga.OpenLigaAccessor
+import de.novatec.betting.game.openliga.model.OLTeam
+import de.novatec.betting.game.teams.model.Team
+import de.novatec.betting.game.teams.model.Teams
+import de.novatec.betting.game.teams.tf.TeamsTf
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkClass
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import utils.classification.UnitTest
+
+val openLigaAccessor: OpenLigaAccessor = mockk()
+val teamsTf: TeamsTf = mockk()
+
+@UnitTest
+class TeamsServiceUnitTest {
+
+    private val service = TeamsService(openLigaAccessor, teamsTf)
+
+    @Test
+    fun `get teams for a season`() {
+
+        val olTeams = listOf(mockkClass(OLTeam::class), mockkClass(OLTeam::class))
+        val teams = mockkClass(Teams::class)
+
+        every { openLigaAccessor.getAllTeams("2020") } returns olTeams
```

Comment:
Hier hätte ich tendenziell mit echten Objekten gearbeitet und nicht mit Mock-Klassen. 
Aber eigentlich macht der Service nicht viel, das Mapping/Transforming ist ja an andere Stelle getestet. Also passt vmtl. auch so.

Reply:
genau, die werden nur durchgereicht. Hatte am Anfang tatsäschlich 'echt' Objekte, aber genau das wird ja wie du sagst in anderen unit test getestet